### ISSUE TITLE
Different `tau` array in qr

### ISSUE THREAD

**Issue by ranjanan** [JuliaComputing/ArrayFire.jl#6]
The array `tau` that's returned looks different from what Julia returns. Why are they different?

**Comment by pavanky** [comment]
@ranjanan I think `tau` is implementation dependent.

**Comment by ranjanan** [comment]
Yes, this is a placeholder for me to check how the `arrayfire` and Julia implementations are different.

**Comment by pavanky** [comment]
I am just saying `tau` is never used by itself so the values shouldn't matter. It is always used along with the other outputs from QR.

**Comment by alanedelman** [comment]
That it's different is not to me as important as whether it's internally consistent.
